x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/get_threat_list.ts

```diff
@@ -54,14 +61,31 @@ export const getThreatList = async ({
   return response;
 };
 
+/**
+ * This returns the sort with a tiebreaker if we find out we are only
+ * querying against the list items index. If we are querying against any
+ * other index we are assuming we are 1 or more ECS compatible indexes and
+ * will query against those indexes using just timestamp since we don't have
+ * a tiebreaker.
+ */
 export const getSortWithTieBreaker = ({
   sortField,
   sortOrder,
+  index,
+  listItemIndex,
 }: GetSortWithTieBreakerOptions): SortWithTieBreaker[] => {
   const ascOrDesc = sortOrder ?? 'asc';
-  if (sortField != null) {
-    return [{ [sortField]: ascOrDesc, '@timestamp': 'asc' }];
+  if (index.length === 1 && index[0] === listItemIndex) {
```

Comment:
I initially created a rule against a threat index like `.items-*`, which does not satisfy this condition and thus still suffers from this issue. This is mentioned in the description as "single index" but I wanted to call it out again since it's easy to miss.

@FrankHassanabad do we want to add a note about this in the threat match rule documentation, or is too much of a "power user" scenario?

Reply:
I think it's a bit of a "power user" scenario and a bit of a trick. We can support it nicely in the UI because we can query for the index list item name and populate it like it is here.

For API users they will have to know their space names ahead of time and kind of fill in the API so we might want to update at least the API docs before release time to show an example of how to do it or we might need to add a better API for when they have a mapping and want to refer to a value based list.